src/ImageSharp/Formats/Webp/Lossless/HuffmanUtils.cs

```diff
@@ -202,9 +202,14 @@ public static void GenerateOptimalTree(HuffmanTree[] tree, uint[] histogram, int
                 }
 
                 // Build the Huffman tree.
+#if NET5_0_OR_GREATER
+                Span<HuffmanTree> treeSlice = tree.AsSpan().Slice(0, treeSize);
+                treeSlice.Sort(HuffmanTree.Compare);
+#else
                 HuffmanTree[] treeCopy = tree.AsSpan().Slice(0, treeSize).ToArray();
```

Comment:
It's unfortunate that .NET Core 3.1 doesn't give us sort here. We could copy [`ArraySortHelper`](https://source.dot.net/#System.Private.CoreLib/ArraySortHelper.cs,63a9955a91f2b37b) or even use something like [`SortUtility`](https://github.com/SixLabors/ImageSharp.Drawing/blob/e987bd8c428792c8a00cce0bc50b59b6401a280c/src/ImageSharp.Drawing/Utilities/SortUtility.cs) from Drawing but I don't know if it's worth the effort. @antonfirsov what do you think?